services/galley/src/Galley/API/Teams.hs

```diff
@@ -385,3 +386,12 @@ finishCreateTeam team owner others zcon = do
     let r = membersToRecipients Nothing others
     push1 $ newPush1 zusr (TeamEvent e) (list1 (userRecipient zusr) r) & pushConn .~ zcon
     pure (empty & setStatus status201 . location (team^.teamId))
+
+getBindingTeamMembers :: UserId -> Galley Response
+getBindingTeamMembers zusr = do
+    tid <- Data.oneUserTeam zusr >>= ifNothing teamNotFound
+    binding <- Data.teamBinding tid >>= ifNothing teamNotFound
+    members <- Data.teamMembers tid
```

Comment:
Perhaps we can switch the order of operations here to avoid a call to `Data.teamMembers` if `binding == NonBinding` ?

Reply:
It won't be evaluated in the NonBinding case afaik

Reply:
fixed